Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -2526,28 +2526,41 @@ private void RestoreDevicesAfterDomainReloadIfNecessary()
             #endif
         }
 
+#if UNITY_EDITOR
         private void SyncAllDevicesWhenEditorIsActivated()
         {
-            #if UNITY_EDITOR
             var isActive = m_Runtime.isEditorActive;
             if (isActive == m_EditorIsActive)
                 return;
 
             m_EditorIsActive = isActive;
             if (m_EditorIsActive)
+                SyncAllDevices();
+        }
+
+        private void SyncAllDevices()
```

Comment:
Any reason that SyncAllDevices isn't internal and used instead of SyncAllDevicesAfterEnteringPlayMode?

Reply:
I think `SyncAllDevices` is way too leaky, like, sync why? And allowing it to be called from other places might have unexpected consequences.
In this case `SyncAllDevicesAfterEnteringPlayMode` is very specific about why, and in the future refactoring we can easier reimplement it some other way, or nuke it all together.
But agree this is all purely syntax sugar.

Reply:
Nice